internal/testplanet/storagenode.go

```diff
@@ -176,7 +177,7 @@ func (planet *Planet) newStorageNodes(count int, whitelistedSatellites storj.Nod
 			return xs, err
 		}
 
-		err = db.CreateTables()
+		err = db.CreateTables(context.TODO())
```

Comment:
Pass in the context from the level above. The annoying thing about `context.TODO` is that somebody has to  later go and fix them. Of course there's a slight problem this introduces as a result. `testplanet.New` assumes there's no context passed in, this is very intentional, to ensure that people don't write code in constructors that run for long time or start lifecycles. That lead to issues with some database tables being used in constructors, before the database has been created. Here the `CreateTables` should probably happen before starting up all the systems, just as it happens in binaries.

Reply:
I believe this is the only place we used ctx.TODO, because it was in test code. We can look and see if we can fix it in another way.

Reply:
Let's create appropriate issues so we don't lose what still needs to be fixed.

Reply:
Yup will do!